### ISSUE TITLE
Does not display the menu MQBCoding.

### ISSUE THREAD

**Issue by aleksgeko** [jilleb/mib2-toolbox#202]
Hi all!
I also read in **closed cases** but for me the information there was not valid.
I want you to help me with a problem with the installation of this MQBCoding script.
I tried many ways such as:
- SWDL - I checked Y and tried to install at least 10 times, with each version, both older and newer.
- I tried another SD card, it didn't work.
- I tried another car, it still didn't work out.

I checked to be:
-SDCard in SD1
- FAT32 format
- 2xSdcard - 1x8GB, 1x32GB.
Tomorrow I will try a different laptop, to the absurd maybe it's because of this.

If you have any other suggestions, please help me.

![WhatsApp Image 2022-02-04 at 21 29 31](https://user-images.githubusercontent.com/81624716/152591426-ecc78952-f96a-48c4-a220-cd1931a09ee4.jpeg)
![WhatsApp Image 2022-02-04 at 21 29 32](https://user-images.githubusercontent.com/81624716/152591431-f08cb9a6-7469-48a4-8995-6c25403300ee.jpeg)

**Comment by fbuser1** [comment]
What is your hardware?

**Comment by aleksgeko** [comment]
Today i tried for another cars, and work very well.
For this car, don't work!

**Comment by NidhalBENJDIRA** [comment]
Hi all,
I have the same problem, install was successful but it didn't display the Menu list.
any advice please.
![image](https://user-images.githubusercontent.com/62646590/165002847-e30b64f7-4a5c-4b2f-abaf-f18d3e787f6e.png)
![image](https://user-images.githubusercontent.com/62646590/165002884-19b1eca1-bbf6-4de9-9124-308889e6c786.png)
![image](https://user-images.githubusercontent.com/62646590/165002916-58b56e23-ed14-48d6-bd00-2b987002e935.png)

thank you.

**Comment by Krosta8** [comment]
I had this issue before too. Solution for me was, that I was installing the toolbox again and again till the menu was correct. I don't know, why it happened to me, for friends it was OK, only for me, I had to install it 3x I think. ŠKODA unit version 1440, not patched (just for reference).

**Comment by NidhalBENJDIRA** [comment]
> I had this issue before too. Solution for me was, that I was installing the toolbox again and again till the menu was correct. I don't know, why it happened to me, for friends it was OK, only for me, I had to install it 3x I think. ŠKODA unit version 1440, not patched (just for reference).

Hi, i tried to re-install but i wasn't enable to click on start button (it's disabled, grey) see picture below.
how to remove the toolbox and install again ? help please
![image](https://user-images.githubusercontent.com/62646590/165161545-49da1ad2-24cb-4265-894f-5ac9f8b7bf95.png)

**Comment by NidhalBENJDIRA** [comment]
Hi every one,

No body can help? at list on the procedure to remove and re-install toolbox again.

Thanks.

**Comment by Krosta8** [comment]
You need to use SWDL and force reinstall

**Comment by NidhalBENJDIRA** [comment]
> You need to use SWDL and force reinstall

Thank you for your feedback.
Can you please detail this step by step, I'm really new to this.

**Comment by Krosta8** [comment]
If I remember correctly, you need to enable development mode to see SWDL option. I can't look on it to do it step by step. In other words, you need some kind of diagnostic for that to enable it - VCDS, VCP, OBDeleven, ODIS and others

**Comment by NidhalBENJDIRA** [comment]
> If I remember correctly, you need to enable development mode to see SWDL option. I can't look on it to do it step by step. In other words, you need some kind of diagnostic for that to enable it - VCDS, VCP, OBDeleven, ODIS and others

It is much clearer, thank you very much for your support

**Comment by olli991** [comment]
https://mibwiki.one/doc/mhi2q-screens-Z6jhPLo51S 
Look into this Screenshots and explanations. You need Testmode and do Swdl and activate the modules you want to force reinstall.

**Comment by NidhalBENJDIRA** [comment]
> https://mibwiki.one/doc/mhi2q-screens-Z6jhPLo51S Look into this Screenshots and explanations. You need Testmode and do Swdl and activate the modules you want to force reinstall.

thank you very much for sharing, i will go through the tutorials to try to find the solution. good day

**Comment by b3ga** [comment]
Anyone managed to sort this thing out?
I tried to update and stuff like that but still not working.Having same menu like [NidhalBENJDIRA]
Throught SWDL can't force reinstall Start is grey. Here is some pictures.

**Comment by olli991** [comment]
Judging your screenshot you aren't in SWDL manual mode because the screen will look like this:
![SCREENSHOT_(Sdl_7DD2(Sdsm;Sgi))_TIME_2000_01_01_00_00_00_GEOCOORD_LAT_48 762_LONG_11 425_4241996_Nr_32](https://user-images.githubusercontent.com/55631413/168620959-55c86c07-3654-40da-b3c0-401e0ec3b9e9.png)
If you have this screen you can go into every module and activate it again from N to Y

You can read more about that here: https://mibwiki.one/doc/mhi2q-screens-Z6jhPLo51S

**Comment by b3ga** [comment]
> Judging your screenshot you aren't in SWDL manual mode because the screen will look like this:
> ![SCREENSHOT_(Sdl_7DD2(Sdsm;Sgi))_TIME_2000_01_01_00_00_00_GEOCOORD_LAT_48 762_LONG_11 425_4241996_Nr_32](https://user-images.githubusercontent.com/55631413/168620959-55c86c07-3654-40da-b3c0-401e0ec3b9e9.png)
> If you have this screen you can go into every module and activate it again from N to Y
> 
> You can read more about that here: https://mibwiki.one/doc/mhi2q-screens-Z6jhPLo51S
> 

Will try later when I get home.When I press longer Menu button there is Green Menu and under it there is SWDL I was there.My menu didnt look like this one on yours picture :)

**Comment by NidhalBENJDIRA** [comment]
Hello @b3ga ,
Here is how to do it.


https://user-images.githubusercontent.com/62646590/168660588-bde654e4-8b12-4b5a-97f4-7b0314e4ae8b.mp4

**Comment by olli991** [comment]
Thanks for the vid. Yes that's the correct way. But keep in mind that on seat the check marks sadly aren't shown correctly. It will work but you have to try it manual download is active or not. You see this if you start the update and have the additional menu above the list, like you can see in my Screenshot.

**Comment by b3ga** [comment]
Thank you for this I managed to install it again manual and it works like a charm.
I did mistake entering in second menu,but I needed to do it in first menu. There is SWDL in another menu,but in first haven't seen in Testmode that SWDL. After reinstall everything is fine.

> Hello @b3ga , Here is how to do it.
> 
>  VID_20220516_182851_2.mp4

Thank you for this I managed to install it again manual and it works like a charm.
I did mistake entering in second menu,but I needed to do it in first menu. There is SWDL in another menu,but in first haven't seen in Testmode that SWDL. After reinstall everything is fine.
Thanks again [olli991](https://github.com/olli991) and [NidhalBENJDIRA](https://github.com/NidhalBENJDIRA)
